lxd/storage/drivers/driver_zfs_utils.go

```diff
@@ -446,3 +446,8 @@ func (d *zfs) datasetHeader(vol Volume, snapshots []string) (*ZFSMetaDataHeader,
 func (d *zfs) randomVolumeName(vol Volume) string {
 	return fmt.Sprintf("%s_%s", vol.name, uuid.New())
 }
+
+func (d *zfs) expandPool(loopDevPath string) error {
+	_, err := shared.RunCommand("zpool", "online", "-e", d.name, loopDevPath)
```

Comment:
Given this is a one line function and only called from one place I think I'd be tempted to move it into the Update function above.